model/src/id/mod.rs

```diff
@@ -281,6 +299,18 @@ impl<T> FromStr for Id<T> {
     }
 }
 
+impl<T> From<Id<T>> for u64 {
+    fn from(id: Id<T>) -> Self {
+        id.get()
+    }
+}
+
+impl<T> From<Id<T>> for NonZeroU64 {
+    fn from(id: Id<T>) -> Self {
+        id.inner()
+    }
+}
+
```

Comment:
Please add tests for these to `test_initializers`

Reply:
Added some tests now 🙇